Configurations/00-base-templates.conf

```diff
@@ -69,11 +77,22 @@ my %targets=(
         ARFLAGS         => "qc",
         CC              => "cc",
         lflags          =>
-            sub { $withargs{zlib_lib} ? "-L".$withargs{zlib_lib} : () },
+            sub { my @libs = ();
```

Comment:
Maybe use the same indentation as the above include block.. It is currently all over the place (same applies to ex_libs)